core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -178,6 +178,9 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
             command.add(runnerJarName);
             //https://github.com/oracle/graal/issues/660
             command.add("-J-Djava.util.concurrent.ForkJoinPool.common.parallelism=1");
+            // org.wildfly.common.net.CidrAddress and org.wildfly.common.net.Inet must be initialized at runtime because of a
+            // new restriction in GraalVM 19.3.0 that prohibits the presence of java.net.Inet4Address in the image heap.
+            command.add("--initialize-at-run-time=org.wildfly.common.net.CidrAddress,org.wildfly.common.net.Inet");
```

Comment:
Should probably use `RuntimeInitializedClassBuildItem` ?

Reply:
That's an interesting suggestion. It won't change the Quarkus module where the runtime initialization is declared though, since `CidrAddress` and `Inet` are only used by the Quarkus configuration code which is in `quarkus-core`. But it still feels better to declare this in a `@BuildStep` than as a command-line option. Thanks!

Reply:
Neither of these classes should be initialized at run time.  What is the original problem?